tests/pipelines/test_pipelines_common.py

```diff
@@ -1907,6 +1916,99 @@ def test_StableDiffusionMixin_component(self):
             )
         )
 
+    @require_torch_multi_gpu
+    @slow
+    @nightly
+    def test_calling_to_raises_error_device_mapped_components(self):
+        if "Combined" in self.pipeline_class.__name__:
```

Comment:
Because for connected pipelines, we don't support device mapping in the first place.